tempesta_fw/classifier/frang.c

```diff
@@ -502,14 +546,70 @@ frang_http_host_check(const TfwHttpReq *req, FrangAcc *ra)
 	return ret;
 }
 
+static unsigned short
+frang_resp_next_idx(unsigned short ab, const unsigned short len)
+{
+	return ++ab >= len ? 0 : ab;
+}
+
+static void
+frang_resp_keep_frame(FrangRespCodeStat *stat, const unsigned long tc)
+{
+	const FrangHttpRespCodeBlock *conf = frang_cfg.http_resp_code_block;
+	const unsigned short ai = (tc % conf->tf) / conf->sf;
+
+	/*
+	 * FrangRespCodeStat is filled with zero in the beginning
+	 * because of TfwClient allocation with __GFP_ZERO flag
+	 */
+	if (tc - stat->tb < conf->tf)
```

Comment:
Don't see how the comment above is connected with the condition. I actually don't see any need in the comment.